deepchecks/suites/default_suites.py

```diff
@@ -0,0 +1,119 @@
+           'model_evaluation', 'full_suite']
+
+
+def single_dataset_integrity() -> Suite:
+    """Create 'Single Dataset Integrity Suite'.
+
+    The suite runs a set of checks that are meant to detect integrity issues within a single dataset.
+    """
+    return Suite(
+        'Single Dataset Integrity Suite',
+        IsSingleValue().add_condition_not_single_value(),
+        MixedNulls().add_condition_different_nulls_not_more_than(),
+        MixedTypes().add_condition_rare_type_ratio_not_less_than(),
+        StringMismatch().add_condition_no_variants(),
+        DataDuplicates().add_condition_ratio_not_greater_than(),
+        StringLengthOutOfBounds().add_condition_ratio_of_outliers_not_greater_than(),
+        SpecialCharacters().add_condition_ratio_of_special_characters_not_grater_than(),
+        LabelAmbiguity().add_condition_ambiguous_sample_ratio_not_greater_than()
+    )
+
+
+def train_test_leakage() -> Suite:
+    """Create 'Train Test Leakage Suite'.
+
+    The suite runs a set of checks that are meant to detect data leakage from the training dataset to the test dataset.
+    """
+    return Suite(
+        'Train Test Leakage Suite',
+        DateTrainTestLeakageDuplicates().add_condition_leakage_ratio_not_greater_than(),
+        DateTrainTestLeakageOverlap().add_condition_leakage_ratio_not_greater_than(),
+        SingleFeatureContribution().add_condition_feature_pps_not_greater_than(),
+        SingleFeatureContributionTrainTest().add_condition_feature_pps_difference_not_greater_than(),
+        TrainTestSamplesMix().add_condition_duplicates_ratio_not_greater_than(),
+        IdentifierLeakage().add_condition_pps_not_greater_than(),
+        IndexTrainTestLeakage().add_condition_ratio_not_greater_than()
+    )
+
+
+def train_test_validation() -> Suite:
+    """Create 'Train Test Validation Suite'.
+
+    The suite runs a set of checks that are meant to validate correctness of train-test split, including
+    integrity, drift and leakage.
+    """
+    return Suite(
+        'Train Test Validation Suite',
+        DominantFrequencyChange().add_condition_ratio_of_change_not_more_than(),
+        CategoryMismatchTrainTest().add_condition_new_category_ratio_not_greater_than(),
+        NewLabelTrainTest().add_condition_new_label_ratio_not_greater_than(),
+        StringMismatchComparison().add_condition_no_new_variants(),
+        TrainTestFeatureDrift().add_condition_drift_score_not_greater_than(),
+        WholeDatasetDrift().add_condition_overall_drift_value_not_greater_than(),
+        DatasetsSizeComparison().add_condition_test_train_size_ratio_not_smaller_than(),
+        train_test_leakage()
+    )
+
+
+def model_evaluation() -> Suite:
+    """Create 'Model Evaluation Suite'.
+
+    The suite runs a set of checks that are meant to test model performance and overfit.
+    """
+    return Suite(
+        'Model Evaluation Suite',
+        ConfusionMatrixReport(),
```

Comment:
start with the general reports (PeformanceReport, TrainTestDifference, Class performance and simple model)